ts/receiver/receiver.ts

```diff
@@ -0,0 +1,824 @@
+// TODO: fix libloki and textsecure not being available here yet
```

Comment:
actually a lot of these should be available if you put them in `window/index.ts`.
Then you can just import like normal.

```
import { libloki, textsecure } from '../window'
```
We're going to be removing `global.d.ts` so using `window.<variable>` won't be allowed.